src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -123,8 +147,40 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
         return Promise.reject(new OrderFinalizationNotRequiredError());
     }
 
+     async verifyCard(methodId: string , amount: number,  payment: any): Promise<GooglePayVerifyPayload>  {
+        if (methodId === PaymentStrategyType.BRAINTREE_GOOGLE_PAY) {
+            const { nonce } = payment.paymentData;
+            const threeDSecure = await this._braintreeSDKCreator?.get3DS();
+
+            return this._braintreePresent3DSChallenge(threeDSecure, amount, nonce);
+        }
+
+    }
+
+    private _braintreePresent3DSChallenge(
+        threeDSecure: BraintreeGooglePayThreeDSecure | undefined,
```

Comment:
Than you can remove some code here:
```suggestion
        threeDSecure: BraintreeGooglePayThreeDSecure,
```

Reply:
fixed